windows/ReactTestApp/ReactTestApp.vcxproj

```diff
@@ -120,7 +124,8 @@
             <AdditionalOptions>%(AdditionalOptions) /bigobj</AdditionalOptions>
             <!--Temporarily disable cppwinrt heap enforcement to work around xaml compiler generated std::shared_ptr use -->
             <AdditionalOptions Condition="'$(CppWinRTHeapEnforcement)'==''">/DWINRT_NO_MAKE_DETECTION %(AdditionalOptions)</AdditionalOptions>
-            <DisableSpecificWarnings></DisableSpecificWarnings>
+            <DisableSpecificWarnings>
+            </DisableSpecificWarnings>
             <PreprocessorDefinitions>WIN32_LEAN_AND_MEAN;WINRT_LEAN_AND_MEAN;REACT_NATIVE_VERSION=10000000;%(PreprocessorDefinitions)</PreprocessorDefinitions>
```

Comment:
I just want to laugh at `WIN32_LEAN_AND_MEAN` 🤣

Reply:
Yup, it's official: https://devblogs.microsoft.com/oldnewthing/20091130-00/?p=15863